generated/nifake/nifake/session.py

```diff
@@ -181,11 +182,12 @@ class _SessionBase(object):
     Example: :py:attr:`my_session.read_write_string_repeated_capability`
     '''
 
-    def __init__(self, repeated_capability_list, all_channels_in_session, library_interpreter, freeze_it=False):
+    def __init__(self, repeated_capability_list, all_channels_in_session, library_interpreter, grpc_channel, freeze_it=False):
         self._repeated_capability_list = repeated_capability_list
         self._repeated_capability = ','.join(repeated_capability_list)
         self._all_channels_in_session = all_channels_in_session
         self._library_interpreter = library_interpreter
+        self._grpc_channel = grpc_channel
```

Comment:
You're right, I don't need it. At least not anymore, in this latest iteration of the code. I store it here for the `_RepeatedCapabilities` class, which needs it so that it can properly construct a new `_SessionBase` object, but now the only place it's used in `_SessionBase` is to store it! I'll remove it.

Reply:
Removed